libraries/AP_HAL_ChibiOS/hwdef/SPEDIXH743/README.md

```diff
@@ -0,0 +1,103 @@
+# SPEDIX H743 Flight Controller
+
+The SPEDIX H743 is a flight controller based on the STM32H743 MCU.
+
+## Features
+
+* MCU - STM32H743 32-bit processor running at 480 MHz
+* IMU - Dual ICM42688
+* Barometer - SPL06
+* OSD - MAX7456
+* 8x UARTs
+* CAN support
+* 9x PWM Outputs (8 Motor Output, 1 LED)
+* Battery input voltage: 2S-6S
+* BEC 3.3V 0.5A
+* BEC 5V 3A
+* BEC 12V 3A for video, gpio controlled
+* Dual camera inputs with switching support
+
+## Pinout
+
+![SPEDIX H743 Board](SpedixH743.png "SPEDIX H743")
+
+## UART Mapping
+
+* SERIAL0 -> USB
+* SERIAL1 -> UART1 (Telem 1, DMA-enabled)
+* SERIAL2 -> UART2 (RC Input, DMA-enabled)
+* SERIAL3 -> UART3 (Telem 2, DMA-enabled)
+* SERIAL4 -> UART4 (MSP DisplayPort, DMA-enabled)
+* SERIAL5 -> UART5 (ESC Telemetry)
+* SERIAL6 -> UART6 (GPS, DMA-enabled)
+* SERIAL7 -> UART7 (Spare)
+* SERIAL8 -> UART8 (Spare)
```

Comment:
Where is this??its not pinned out that I can see....maybe you meant to define the OTG2 (SCLCAN) port?

Reply:
Changed to OTG2